narayana/ArjunaJTA/jta/pom.xml

```diff
@@ -139,6 +139,17 @@
                 <target>1.8</target>
               </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
```

Comment:
This is something which you really don't want to do. Two definitions of one plugin under `<build>` tag. The top level elements overload each other. The last declared wins (I assume). But I don't expect it's desired (while it's confusing).

Reply:
I will remove these changes, as per Mike's comments below